pyiceberg/transforms.py

```diff
@@ -146,7 +146,13 @@ def can_transform(self, source: IcebergType) -> bool:
         return False
 
     @abstractmethod
-    def result_type(self, source: IcebergType) -> IcebergType: ...
+    def result_type(self, source: IcebergType) -> IcebergType:
+        """
+        Return the type of the result of the transform.
```

Comment:
```suggestion
        Returns the `IcebergType` produced by this transform given a source type.
```

https://github.com/apache/iceberg/blob/main/api/src/main/java/org/apache/iceberg/transforms/Transform.java#L71-L77